dev/io.openliberty.grpc.1.0.internal.client/bnd.bnd

```diff
@@ -15,9 +15,10 @@
 -sub: *.bnd
 
 bVersion=1.0
-grpcVersion=1.43.2
+grpcVersion=1.57.2
 
 Import-Package: \
+  !com.google.errorprone.*,\
```

Comment:
This is a new package for this version of netty libs?

Reply:
Yes sort of - it is actually used in the most recent Google gRPC Java code but is a compile time only Annotation so is not needed at runtime and is contained in a different jar/dependency. If this line is not there (in each of the 3 places) then there is an error.